### PR TITLE
Fix __dump_task_bounding_box

### PR DESCRIPTION
**Pull request by daniel-wer**
The call to `__dump_bounding_box` was missing the first parameter. A linter should've catched this, but since we're moving this code to webknossos-libs, it's probably not worth it to introduce one at this point.

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -332,7 +332,7 @@ version = "1.17.4"
 description = "NumPy is the fundamental package for array computing with Python."
 category = "main"
 optional = false
-python-versions = ">=3.6"
+python-versions = ">=3.5"
```
Comment (hotzenklotz) [inline poetry.lock:335]: Is this change on purpose?

File: wknml/__init__.py [hunk 2]
```diff
@@ -437,7 +437,7 @@ def parse_nml(file: BinaryIO) -> NML:
 def __dump_task_bounding_box(xf: XmlWriter, parameters: NMLParameters):
     task_bounding_box = getattr(parameters, "taskBoundingBox")
     if task_bounding_box is not None:
-        __dump_bounding_box(task_bounding_box, "taskBoundingBox")
+        __dump_bounding_box(xf, task_bounding_box, "taskBoundingBox")
```
Comment (hotzenklotz) [inline wknml/__init__.py:440]: Good catch. Thanks.

**Review by hotzenklotz — APPROVED** [review]

File: poetry.lock (hunk 1 above)
Comment (daniel-wer) [reply poetry.lock:335]: As discussed in Slack: Yes this was done automatically by poetry after running `poetry lock --no-update` to fix the hashes. Previously the lock file contained the hashes for the wrong numpy version.
